solr/core/src/java/org/apache/solr/handler/StreamHandler.java

```diff
@@ -191,7 +237,20 @@ public void inform(SolrCore core) {
       .withFunctionName("lteq", LessThanEqualToEvaluator.class)
       .withFunctionName("not", NotEvaluator.class)
       .withFunctionName("or", OrEvaluator.class)
-      
+
+      // Date Time Evaluators
+      .withFunctionName(TemporalEvaluatorYear.FUNCTION_NAME, TemporalEvaluatorYear.class)
```

Comment:
I'd like to approach this in a separate ticket. Would it be alright if we change the use of TemporalEvaluatorYear.FUNCTION_NAME to "year" in the call to `withFunctionName` for now?